src/Stryker.Core/Stryker.Core/Mutators/LinqMutator.cs

```diff
@@ -33,23 +34,25 @@ static LinqMutator()
                 { LinqExpression.TakeWhile, LinqExpression.SkipWhile },
                 { LinqExpression.Min, LinqExpression.Max },
                 { LinqExpression.Max, LinqExpression.Min },
-                { LinqExpression.Sum, LinqExpression.Count },
+                { LinqExpression.Sum, LinqExpression.Max },
                 { LinqExpression.Count, LinqExpression.Sum },
+                { LinqExpression.Average, LinqExpression.Min },
                 { LinqExpression.OrderBy, LinqExpression.OrderByDescending },
                 { LinqExpression.OrderByDescending, LinqExpression.OrderBy },
                 { LinqExpression.ThenBy, LinqExpression.ThenByDescending },
-                { LinqExpression.ThenByDescending, LinqExpression.ThenBy }
+                { LinqExpression.ThenByDescending, LinqExpression.ThenBy },
+                { LinqExpression.Reverse, LinqExpression.AsEnumerable },
```

Comment:
I think it might be better to just remove the Reverse method call.

Reply:
Remove Reverse would do the trick, indeed, but it would require special, and non trivial, logic to deal with Reverse. Using AsEnumerable as a no-op seems better suited for me.